wordpress-mobile/WordPress-iOS
# Login: fix wpcom password placement with on-screen keyboard

The new wpcom password screen gets slightly weird layout with the on screen keyboard.

Before | After
----|----
![wpcom-fix-before](https://user-images.githubusercontent.com/517257/27745905-6668582e-5d82-11e7-9f73-b0ff79e39112.gif)|![wpcom-fix-after](https://user-images.githubusercontent.com/517257/27745920-6f674926-5d82-11e7-9257-14581a432a9c.gif)

To test:
- make sure wpcom login via password still works 😜
- ensure the placement of the password field works well

### Note:
this was branched off #7465 and will show its changed until they're merged.

Needs review: @aerych
